wix-data/wix-data/WixDataFilter.service.json

```diff
@@ -517,8 +519,8 @@
                 "",
                 " Matching strings with `eq()` is case sensitive, so `\"text\"` is not equal to `\"Text\"`.",
                 "",
-                " If the value of the `propertyName` property is an Array, `eq()` includes items",
-                " in which any of the elements of the Array match the specified `value`." ],
+                " If `propertyName` points to a collection field of type Array, `eq()` includes all items",
+                " in which at least one Array element matches the specified `value`." ],
```

Comment:
Maybe it would be a bit clearer this way?
"If `propertyName` points to a collection field of type Array, `eq()` includes it as long as at least one Array element matches the specified `value`."

Reply:
I'd be more specific than "it". What do you think of:
"If `propertyName` points to a collection field of type Array, `eq()` includes the item as long as at least one Array element matches the specified value."

Reply:
Fair point about "it". I wonder if people will be confused by the terms "collection field" and "item" when "item" is really referring to the contents of the collection field.

I feel like I am nitpicking at this point though. So if you are not worried about it, go with what you have here.